cfgov/jinja2/v1/_includes/organisms/mega-menu.html

```diff
@@ -193,5 +193,13 @@
     {# Create a root menu at depth one.
        This is the 1st level of a 3-level menu. #}
     {{ _nav( 1, vars.nav_groups ) }}
-    <button class="{{ base_class ~ '_tab-trigger' }}" aria-hidden="true"></button>
+    <button class="m-global-search_tab-trigger u-visually-hidden"
+            aria-hidden="true">
+      .
```

Comment:
Change text to read "Collapse" instead of ".", and adjust the code comment describing it.

Reply:
Fixed